### DISCUSSION TITLE
透明网关无法代理群晖NAS

### DISCUSSION THREAD

**Discussion by elfzweik** [XTLS/Xray-core#693] [Q&A]
采用Linux服务器架设透明网关的方式，即主路由192.168.0.1的DHCP网关指向Linux服务器192.168.0.8，Linux服务器的网关指向主路由192.168.0.1，XRAY跑在Linux服务器上。家里的其他设备都可以通过XRAY上网了，突然发现群晖NAS没有被代理。表现如下：
1. 群晖NAS启动video station，测试刮削插件The movie database，连接失败。
2. 检查Linux服务器上的XRAY日志文件，发现没有监听到群晖NAS（192.168.0.117）的请求。

尝试将群晖NAS的网关和DNS服务器手动指向Linux服务器192.168.0.8，设置后无法通过ddns:port方式访问群晖，quickconnect还可以访问，此时video station的刮削插件可以连通The movie database，刮削正常。所有端口转发规则因网关改变而失效。重新设置upnp规则时，路由器设置通不过，网络环境检测失败和路由器不能识别，无法重置转发规则。通过`iptables -t mangle -I XRAY_SELF -s 192.168.0.0/16 -p tcp --sport 5001 -j RETURN`也无法使用5001端口访问群晖。
问题1：主路由的DHCP网关下发为什么对群晖失效?
问题2：由此还联想到，在没有改动群晖的网关时，为了外网SSH透明网关或群晖，@bioniCosmos 需要设置转发规则`iptables -t mangle -I XRAY_SELF -s 192.168.0.0/16 -p tcp --sport 22 -j RETURN`使22端口的流量不经过XRAY，为什么其他端口未设置这样的转发规则，仍然能使用，比如jellyfin、transmission的端口都能正常使用。

求指点，谢谢！

**Comment by drunkleee** [comment]
绑定

**Reply by elfzweik** [reply]
麻烦说详细一点，绑定什么？
